stack.yaml

```diff
@@ -1,8 +1,12 @@
-resolver: lts-9.11
+resolver: lts-12.3
 packages:
 - .
 extra-deps:
-- 'quickcheck-classes-0.1'
+# - 'quickcheck-classes-0.1'
```

Comment:
Just delete this line instead of commenting it out

Reply:
oops, my bad. done